service/src/main/java/com/fecklessweasel/service/UniversityServlet.java

```diff
@@ -21,6 +21,7 @@
 	 * of universities given a set of parameters.
 	 * **/
 	String longName;
+	String state;
```

Comment:
Any reason this is an instance variable instead of a local variable? Also, it's package protected and accessible outside this class.